dubbo-admin/dubbo-ops-frontend/src/components/ServiceDetail.vue

```diff
@@ -0,0 +1,220 @@
+            <td>{{props.item.role}} </td>
+            <td>{{props.item.value}}</td>
+          </template>
+        </v-data-table>
+      </v-flex>
+      <v-flex sm12>
+        <h3>service info</h3>
+      </v-flex>
+      <v-flex lg12 >
+        <v-tabs
+          class="elevation-1">
+          <v-tab>
+            providers
+          </v-tab>
+          <v-tab>
+            consumers
+          </v-tab>
+          <v-tab-item>
+            <v-data-table
+              class="elevation-1"
+              :headers="detailHeaders.providers"
+              :items="providerDetails"
+            >
+              <template slot="items" slot-scope="props">
+                <td>{{props.item.ip}}</td>
+                <td>{{props.item.port}}</td>
+                <td>{{props.item.timeout}}</td>
+                <td>{{props.item.serial}}</td>
+                <td>URL</td>
+              </template>
+            </v-data-table>
+          </v-tab-item>
+          <v-tab-item >
+            <v-data-table
+              class="elevation-1"
+              :headers="detailHeaders.consumers"
+              :items="consumerDetails"
+            >
+              <template slot="items" slot-scope="props">
+                <td>{{props.item.ip}}</td>
+                <td>{{props.item.port}}</td>
+                <td>{{props.item.appName}}</td>
+              </template>
+            </v-data-table>
+          </v-tab-item>
+        </v-tabs>
+      </v-flex>
+      <v-flex sm12>
+        <h3>meta data</h3>
+      </v-flex>
+      <v-flex lg12>
+        <v-data-table
+          class="elevation-1"
+          :headers="metaHeaders"
+          :items="metadata">
+          <template slot="items" slot-scope="props">
+            <td>{{props.item.method}}</td>
+            <td>{{props.item.parameter}}</td>
+            <td>{{props.item.returnType}}</td>
+          </template>
+        </v-data-table>
+      </v-flex>
+    </v-layout>
+  </v-container>
+</template>
+<script>
+  export default {
+    props: {
+      basic: {
+        type: Array,
+        default: () =>
+          [
+            {
+              role: '应用名',
+              value: 'dubbo-demo'
+            },
+            {
+              role: '服务名',
+              value: 'dubbo.com.alibaba'
+            },
+            {
+              role: 'xxxx',
+              value: 'ffffff'
+            }
+          ]
+      },
+      providerDetails: {
+        type: Array,
+        default: () =>
+          [
+            {
+              ip: '192.168.0.1',
+              port: '28880',
+              timeout: '3000',
+              serial: 'hessian'
+            },
+            {
+              ip: '192.168.0.8',
+              port: '28880',
+              timeout: '3000',
+              serial: 'hessian'
+
+            }
+          ]
+      },
+      consumerDetails: {
+        type: Array,
+        default: () =>
+          [
+            {
+              ip: '192.168.1.3',
```

Comment:
I don't have experience in `vue.js`, so I may wrong here. but this block looks like some test data? is it a right place to include it ?

Reply:
it's the default value for test purpose